building_applications/building_side_panel_apps.md

```diff
@@ -279,6 +264,27 @@ The following sections list the currently supported Side Panel Views that can be
 | /:project_id/project/calendar/lookaheads | schedule.calendar.lookaheads.list |
 | /:project_id/project/calendar/tasks/:task_id | schedule.calendar.task.detail |
 
+### 5. Save the New Component
+
+* Once you have successfully created a new side panel component and defined the Type, Description, URL, Parameter(s), and View(s), click **Save Component**.
+
+  ![Component Save]({{ site.baseurl }}/assets/guides/form-based-component-save.png)
+
+<a name="define-setup-instructions"></a>
+{% include setup_instructions.md %}
+
+## Save Manifest and Create Version
+
+After completing the steps above, you're ready to save your app's configurations and set your first version number. 
+1. Click **Save** at the top of the page.
+2. The **Create Version** window appears. Enter a version number using the following syntax: **x.x.x.**
+Note: Your version number can only contain integers, and must consist of three (3) integers separated by a '.'. For example, '1.1.1'. Each new version you create must be a higher number than the previous version.
+
+## Promote Updated Sandbox Manifest to Production
+
+After you have successfully tested and validated your updated manifest in the development sandbox environment, you can promote it to the production environment.
```

Comment:
It may be worthwhile to add a 'How to install your app in the development sandbox environment' to multiple steps on this page.

This specific line is a great example - we are telling the end-user to go test and validate the updated manifest, but the immediate section above only details how to save and create a version. In this case, the end-user could make an assumption that the newly created app manifest is automatically installed, which is NOT the case. Same for updates to the app manifest - this requires a new installation to see the updated changes to the manifest.

This comment applies to multiple sections - such as "Steps for creating a side panel application", and generally, all applications.

Reply:
General comment, not sure which section this pertains to, but I think it'll be helpful to show documentation on what the install process would look like from a customers perspective and highlight how post installation notes/url show up in a customer's view. A lot of developers don't know that some of the info is public facing on the Marketplace page or in a customer's App Managements